app/src/index.tsx

```diff
@@ -1,14 +1,14 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.scss";
```

Comment:
What kind of rule is replacing the double quotes with single ones?

Reply:
![image](https://user-images.githubusercontent.com/28546954/157468984-41727ccb-9086-4188-84ba-74319ecb0edd.png)

that would be Prettier 
should I set it on double quotes?